compiler/driver/src/parser/queries.rs

```diff
@@ -153,7 +162,7 @@ where
     Ok(new_module)
 }
 
-pub fn find_sources<D, P>(db: &D, dir: P) -> anyhow::Result<Arc<Seq<InternedInput>>>
+pub fn find_sources<D, P>(db: &D, dir: P) -> anyhow::Result<Vec<InternedInput>>
```

Comment:
What was the motivation behind this change?

Reply:
It just made it cleaner for the use of line 64 and 64 above.